src/PaymentServer/Server.hs

```diff
@@ -36,15 +40,15 @@ type PaymentServerAPI
   :<|> "v1" :> "redeem" :> RedemptionAPI
 
 -- | Create a server which uses the given database.
-paymentServer :: VoucherDatabase d => Issuer -> d -> Server PaymentServerAPI
-paymentServer issuer database =
-  stripeServer database
+paymentServer :: VoucherDatabase d => ByteString -> Issuer -> d -> Server PaymentServerAPI
```

Comment:
Perhaps a type alias of `StripeSecretKey` for `ByteString` would help readability?

Reply:
Yes, for sure.